snmp/datadog_checks/snmp/data/profiles/palo-alto.yaml

```diff
@@ -15,7 +24,53 @@ extends:
 device:
   vendor: "paloaltonetworks"
 
-sysobjectid: 1.3.6.1.4.1.25461.*
+# All Palo Alto Networks sysObjectID start with `1.3.6.1.4.1.25461.2.3` (panMibs)
+sysobjectid: 1.3.6.1.4.1.25461.2.3.*
+
+# Note related to use of index `1` of entPhysicalTable columns:
+# Based on snmpwalks of Palo Alto Networks devices, the main/chassis hardware have the index `1`
+# Librennms is also using index `1`: https://github.com/librenms/librenms/blob/1ac60e3b1d90616119f3c4adc28213e3c35c2477/includes/definitions/discovery/arista_eos.yaml#L4
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "paloaltonetworks"
+      serial_number:
+        symbol:
+          OID: 1.3.6.1.4.1.25461.2.1.2.1.3.0
+          name: panSysSerialNumber # The serial number of the unit. If not available, an empty string is returned.
+      version:
+        symbol:
+          OID: 1.3.6.1.4.1.25461.2.1.2.1.1.0
+          name: panSysSwVersion # Full software version. The first two components of the full version are the major
+                                # and minor versions. The third component indicates the maintenance release number
+                                # and the fourth, the build number.
+      product_name:
+        symbol:
+          OID: 1.3.6.1.2.1.1.1.0
+          name: sysDescr
+          match_pattern: 'Palo Alto Networks\s+(PA-\d+ series firewall|WildFire Appliance|VM-Series firewall)'
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Have you considered catching anything after `Palo Alto Networks` instead of trying to match with hardcoded names ?

Reply:
Yes, thought about that, but I preferred to be conservative for now since we don't have many cases and I didn't wanted to risk capturing unwanted things.